tests/functional/test_stacks_config_prototype.py

```diff
@@ -16,10 +16,10 @@
 
 import allure
 import pytest
-from adcm_client.objects import ADCMClient
 from adcm_pytest_plugin import utils
 
 # pylint: disable=redefined-outer-name, protected-access
```

Comment:
now there is no protected access, you can remove this inspection

Reply:
yay
removed